### PR TITLE
chore: update gatsby theme carbon

### PR DESCRIPTION
**Pull request by jnm2377**

### CODE REVIEW

**Review by emyarod — COMMENTED** [review]
I don't have permission to redeploy the preview, would you mind redeploying it on your end?

image

**Review by emyarod — COMMENTED** [review]
it looks like the package upgrade is causing build issues?

File: package.json [hunk 1]
```diff
@@ -45,32 +45,42 @@
...
+    "gatsby-theme-carbon": "^1.27.0",
+    "lodash-es": "^4.17.15",
+    "markdown-it": "^9.0.1",
+    "nanoid": "^2.1.11",
+    "prettier-config-carbon": "^0.4.0",
+    "prop-types": "^15.7.2",
     "react": "^16.8.6",
-    "react-copy-to-clipboard": "^5.0.1"
+    "react-copy-to-clipboard": "^5.0.1",
+    "react-live": "^2.2.1"
```
Comment (jnm2377) [inline package.json:72]: for reviewers, eslint errors said these should be in `dependencies` not `devDependencies`:
Screen Shot 2020-10-29 at 3 24 52 PM

**Comment by jnm2377** [comment]
@emyarod updated the PR. Should be working now. I also had to make some changes to color tokens due to the new theming stuff that we added to the gatsby theme (for context on those changes).

File: src/styles/_homepage.scss [hunk 2]
```diff
@@ -72,3 +72,12 @@
...
 }
+
+// OVERRIDES
+.resource-card-group .bx--resource-card--dark {
+  border-color: #000;
+}
+
+.resource-card-group .bx--col-lg-4:nth-child(even) .bx--resource-card--dark {
+  border-left-color: #000;
+}
```
Comment (jnm2377) [inline src/styles/_homepage.scss:83]: This is a temporary overrride. These styles are coming from the GTC, so they need to be fixed there.

**Review by emyarod — APPROVED** [review]
looks good to me

**Review by joshblack — APPROVED** [review]

**Pull request merged**